src/utils/isOutsideClick.js

```diff
@@ -12,7 +12,8 @@ const getPath = e => {
   return path
 }
 
-export default e => {
+export default (e, className) => {
```

Comment:
Not sure what this has to do with the empty children check. I've a feeling this is either test stuff or you're trying to solve a different issue - in which case it should be separate discussion. Can you elaborate on this please?

Reply:
Yeah this is a separate issue. Do you want me to create another issue for it?

Reply:
When we have more than one dropdowns on the page then if we open one dropdown and then open another, the first one doesn't close because we are checking the `outsideClick` based on CSS class and since both have same classes the dropdown doesn't close on opening another one. And if you click outside then all the dropdown closes at once.


If you want, I can create a separate issue and PR for this.

Reply:
Sounds good! Will merge it soon once I verify locally.

Reply:
While testing this particular change, I found that this works only if the user assigns a unique classname to each instance. Otherwise, it's the same behavior.

Reply:
Right. So it keeps the existing behavior and also gives the user a way to modify it.

Reply:
That wasn't the point I was trying to make but I like the way you put it! :)